assets/js/googlesitekit/data/create-notifications-store.js

```diff
@@ -72,9 +67,29 @@ export const createNotificationsStore = ( type, identifier, datapoint, {
 		// object so we can know if a client notification was added and then
 		// removed from state.
 		clientNotifications: client ? undefined : {},
-		isFetchingNotifications: false,
 	};
 
+	const fetchGetNotificationsStore = createFetchStore( {
```

Comment:
Seeing this in action really illustrates the value of the new approach. Seeing these kind of diff is awesome.

The new `createFetchStore` code is still quite readable for how much is abstracted away, which is impressive. I'm happy we discussed the API so much previous to the review, because this is sooooo much more succinct without being opaque.

All that is to say: great job on this. 🎉